.gitignore

```diff
@@ -1,13 +1,35 @@
-# IntelliJ
-.idea
-*.sqlite3
+# Python gitignore recommandations from https://github.com/github/gitignore/blob/master/Python.gitignore
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# Django stuff:
+*.log
+local_settings.py
```

Comment:
FYI. These files were generated (not ignored) during the python scripts for the DB @peterbogdan 

![image](https://user-images.githubusercontent.com/14240398/94400975-33135c00-016a-11eb-98df-e9375a50ff60.png)

Reply:
well these are migrations so they should not be ignored